frontend/src/features/videos/VideoList.tsx

```diff
@@ -26,7 +26,7 @@ function VideoList({
   const { isLoggedIn } = useLoginState();
 
   const defaultActions = isLoggedIn
-    ? [CompareNowAction, AddToRateLaterList]
+    ? [CompareNowAction, AddToRateLaterList, AddDetailedConsultation]
```

Comment:
We want the analysis page to ba a public page accessible even to users without loggin. For this you should also add the action in the line below:

```TypeScript
isLoggedIn
    ? [CompareNowAction, AddToRateLaterList, AddDetailedConsultation]
    : [AddDetailedConsultation]
```